src/test/java/com/coreos/jetcd/internal/impl/LoadBalancerTest.java

```diff
@@ -0,0 +1,63 @@
+package com.coreos.jetcd.internal.impl;
+
+import com.coreos.jetcd.ClientBuilder;
+import com.coreos.jetcd.KV;
+import com.coreos.jetcd.kv.PutResponse;
+import io.grpc.PickFirstBalancerFactory;
+import io.grpc.util.RoundRobinLoadBalancerFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+/**
+ * KV service test cases.
+ */
+public class LoadBalancerTest {
+
+  @Test
+  public void testPickFirstBalancerFactory() throws Exception {
+    KV client = ClientBuilder.newBuilder()
+        .setEndpoints(TestConstants.endpoints)
+        .setLoadBalancerFactory(PickFirstBalancerFactory.getInstance())
+        .build()
+        .getKVClient();
+
+    PutResponse response;
+    long lastMemberId = 0;
+
+    for (int i = 0; i < TestConstants.endpoints.length * 2; i++) {
+      response = client.put(TestUtil.randomByteSequence(), TestUtil.randomByteSequence()).get();
+
+      if (i == 0) {
+        lastMemberId = response.getHeader().getMemberId();
+      }
+
+      Assert.assertEquals(lastMemberId, response.getHeader().getMemberId());
```

Comment:
I imported [assertJ](http://joel-costigliola.github.io/assertj/) to jetcd for previous testing. I like the fluent style and it is very good for testing exception handling.
e.g
`assertThat(getResponse.getHeader().getMemberId()).isEqualTo(lastMemberId);`